toolkit/tools/imagecustomizerapi/partitiontype.go

```diff
@@ -23,15 +23,96 @@ const (
 	//
 	// See, https://en.wikipedia.org/wiki/BIOS_boot_partition
 	PartitionTypeBiosGrub PartitionType = "bios-grub"
+
+	PartitionTypeHome         = "home"
+	PartitionTypeLinuxGeneric = "linux-generic"
+	PartitionTypeRoot         = "root"
+	PartitionTypeRootVerity   = "root-verity"
+	PartitionTypeSrv          = "srv"
+	PartitionTypeSwap         = "swap"
+	PartitionTypeTmp          = "tmp"
+	PartitionTypeUsr          = "usr"
+	PartitionTypeUsrVerity    = "usr-verity"
+	PartitionTypeVar          = "var"
+	PartitionTypeXbootldr     = "xbootldr"
+)
+
+var (
+	partitionTypeToUuidArchIndependent = map[PartitionType]string{
+		PartitionTypeESP:      "c12a7328-f81f-11d2-ba4b-00a0c93ec93b",
+		PartitionTypeBiosGrub: "21686148-6449-6E6F-744E-656564454649",
+		PartitionTypeHome:     "933ac7e1-2eb4-4f13-b844-0e14e2aef915",
+		PartitionTypeSrv:      "3b8f8425-20e0-4f3b-907f-1a25a76f98e8",
+		PartitionTypeSwap:     "0657fd6d-a4ab-43c4-84e5-0933c84b4f4f",
+		PartitionTypeTmp:      "7ec6f557-3bc5-4aca-b293-16ef5df639d1",
+		PartitionTypeVar:      "4d21b016-b534-45c2-a9fb-5c16e091fd2d",
+		PartitionTypeXbootldr: "bc13c2ff-59e6-4262-a352-b275fd6f7172",
+	}
+
```

Comment:
could add a link to where the uuids come from

Reply:
Fixed.